sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ece-request/models/ece.model.ts

```diff
@@ -0,0 +1,8 @@
+export class ECEProcessResultQueueOutDTO {
```

Comment:
These models can be changed to interfaces, right?

Reply:
This file has only `ECEProcessResult` that has no references. The files can be removed now.